src/hashTable.js

```diff
@@ -0,0 +1,125 @@
+class Node {
+  constructor(key, value) {
+    this.key = key,
+    this.value = value,
+    this.next = null
+  }
+}
+
+export default class HashTable {
+  constructor() {
+    this._length = 0
+    this.valueStore = []
+  }
+
+  hashFunction(key) {
+    let sum = 0
```

Comment:
true, but I wasn't sure if that was allowed since we are supposed to be breaking down the creation of these data structures. Thanks for suggesting that! 👍